plugins/language_cue.lua

```diff
@@ -0,0 +1,42 @@
+-- mod-version:3 
+-- lite-xl 2.0
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "Cue",
```

Comment:
I wonder if this should be uppercase.

Reply:
You are right, `CUE` should be uppercase, and now I reformatted  `language_edp.cue` using the lsp and should be fine.